### PR TITLE
server: Drop unused `WSChiaConnection.is_feeler`

### PR DESCRIPTION
**Pull request by xdustinface**

### CODE REVIEW

**Comment by arvidn** [comment]
I'm pretty sure this used to be used. at least in the crawler. Do you know of a recent patch making this unused?

**Comment by xdustinface** [comment]
> I'm pretty sure this used to be used. at least in the crawler. Do you know of a recent patch making this unused?

Looking at 1.1.3 and 1.3.4 there was no usage of this either.

File: chia/server/server.py [hunk 1]
```diff
@@ -480,7 +479,6 @@ async def start_client(
                 self._port,
                 self.log,
                 True,
-                False,
```
Comment (arvidn) [inline chia/server/server.py:483]: I imagine the original intention was to pass `is_feeler` here

**Review by arvidn — APPROVED** [review]
I see. The parameter to `start_client()` is primarily used to initiate a `close()` task immediately after the connection is established. I imagine the original idea was for the connection to be treated differently in some cases if it was about to be disconnected, but it doesn't appear to be

**Pull request merged**